src/Uno.Extensions.Reactive/Collections/Facades/Differential/Empty.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections;
+using System.Linq;
+
+namespace Uno.Extensions.Collections.Facades.Differential;
+
+/// <summary>
+/// A node of a linked stack of <see cref="IDifferentialCollectionNode"/> which is clears the collection
+/// </summary>
+internal sealed class Empty : IDifferentialCollectionNode
```

Comment:
This looks conflict-y :)

```suggestion
internal sealed class EmptyNode : IDifferentialCollectionNode
```